students/C_Farris/session04/file_lab.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env Python3
+
+"""Date: November 6, 2018
+   Created by: Carol Farris
+   Name: File lab
+   Goal: Learn file handling with Python
+   Remaining work:
+      manipulate the content in the file:
+      take in one line at a time as a long string
+      split the file in two by colons
+      make everything lowercase
+      next, split the sub string by tabs
+"""
+import os
+OUT_PATH = "practice_file_writing"
+
+
+def send_file_to_disk(myfile):
+    """
+    Accepts test file and writes to disk
+    :param (myfile):
+    :return: none. Will send file to directory set in outpath.
+    """
+    prepare_to_write_to_disk()
+    os.chdir(OUT_PATH)
+    with open('practiceFile.txt', 'w') as outfile:
```

Comment:
note that the default is tp open the file in text mode -- so this may fail for non-text files.